packages/components/slider/__tests__/slider.test.tsx

```diff
@@ -32,14 +32,9 @@ describe('Slider', () => {
 
   it('should not exceed min and max', async () => {
     const wrapper = mount({
-      template: `
-        <slider v-model="value" :min="50"></slider>
-      `,
-      components: { Slider },
-      data() {
-        return {
-          value: 60,
-        }
+      data: () => ({ value: 60 }),
```

Comment:
Maybe we can use a ref instead of `data` property
```ts
const value = ref(60)
```

Reply:
updated.